cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/NotificationFeedHttpHandler.java

```diff
@@ -63,13 +63,19 @@ public NotificationFeedHttpHandler(Authenticator authenticator, NotificationFeed
   }
 
   @PUT
-  @Path("/feeds/{id}")
-  public void createFeed(HttpRequest request, HttpResponder responder, @PathParam("id") String id) {
+  @Path("/feeds/categories/{feed-category}/names/{feed-name}")
+  public void createFeed(HttpRequest request, HttpResponder responder,
+                         @PathParam("namespace-id") final String namespaceId,
```

Comment:
They don't need to, but that's the convention I've found in other handlers though.